<launch>
    <node name="pso_slam_node_front" pkg="pso_slam" type="pso_slam" output="screen">
        <!-- <rosparam param="og_cell_side">0.01</rosparam>   -->
        <!-- <rosparam param="odom_topic">"odom"</rosparam>   -->
        <!-- <rosparam param="map_size">25</rosparam>         -->
        <!-- <rosparam param="rate">10</rosparam>             -->
        <!-- <rosparam param="cell_side">0.5</rosparam>       -->
        <rosparam param="num_threads">4</rosparam>
        <rosparam param="iterations">30</rosparam>
        <rosparam param="frame_size">50</rosparam>
        <rosparam param="scan_topic">"/scan_front"</rosparam>
        <rosparam param="sync_topic">"/scan_back"</rosparam>
        <rosparam param="scan_frame">"lidar_front"</rosparam>
        <remap from="/pso_slam/pose" to="/pso_slam/pose_front"/> 
        <!-- <remap from="~odom" to="/odom"/>                 -->
    </node>
</launch>
